src/panels/config/automation/condition/ha-automation-condition-row.ts

```diff
@@ -285,6 +294,31 @@ export default class HaAutomationConditionRow extends LitElement {
     }
   }
 
+  private async _renameCondition(): Promise<void> {
+    const describedCondition = describeCondition(this.condition);
+    const alias = await showPromptDialog(this, {
+      title: this.hass.localize(
+        "ui.panel.config.automation.editor.conditions.change_alias"
+      ),
+      inputLabel: this.hass.localize(
+        "ui.panel.config.automation.editor.conditions.alias"
+      ),
+      inputType: "string",
+      defaultValue: this.condition.alias || describedCondition,
```

Comment:
The `describedCondition`, can we put that as a placeholder? We wouldn’t want people to open rename, then hit ok without doing anything and then ending up with an alias set to the current description.

Reply:
> The describedCondition, can we put that as a placeholder?

I don't believe the prompt dialog supports that, will take a look (or else add it).

>  then hit ok without doing anything and then ending up with an alias set to the current description.

That case is handled a couple of lines below.

Reply:
Added support for placeholders.